terraform/environments/electronic-monitoring-data/share_analytical_platform.tf

```
@@ -457,7 +457,13 @@ resource "aws_iam_role_policy_attachment" "analytical_platform_share_policy_atta
 }
 
 resource "aws_lakeformation_data_lake_settings" "lake_formation" {
-  admins = flatten([[for share in local.analytical_platform_share : aws_iam_role.analytical_platform_share_role[share.target_account_name].arn], [for share in local.analytical_platform_share : aws_iam_role.analytical_platform_share_role[share.assume_account_name].arn], data.aws_iam_session_context.current.issuer_arn, try(one(data.aws_iam_roles.data_engineering_roles.arns), [])])
+  admins = flatten([
+    [for share in local.analytical_platform_share : aws_iam_role.analytical_platform_share_role[share.target_account_name].arn],
+    data.aws_iam_session_context.current.issuer_arn,
+    try(one(data.aws_iam_roles.data_engineering_roles.arns),
+      [for share in local.analytical_platform_share : "arn:aws:iam::${share.assume_account_id}:role/GlobalGitHubActionAdmin"],
+    [])]
+  )
```

```suggestion
  admins = flatten([
    [for share in local.analytical_platform_share : aws_iam_role.analytical_platform_share_role[share.target_account_name].arn],
    data.aws_iam_session_context.current.issuer_arn,
    try(
      one(data.aws_iam_roles.data_engineering_roles.arns),
      [for share in local.analytical_platform_share : "arn:aws:iam::${share.assume_account_id}:role/GlobalGitHubActionAdmin"],
      []
    )
  ])
```
just playing balancing brackets and indents!